### ISSUE TITLE
TokenSet type is incorrect

### ISSUE THREAD

**Issue by cpetzold** [edlink/edlink-node-sdk#13]
It looks like `TokenSet` may not be up to date with the latest 2.0 API response. `expires_in` instead of `expiration_date`, `token_type` instead of `type`, and `context` is missing.

Here's what I'm getting with `grant`:
```json
  {
    "access_token": "...",
    "refresh_token": "...",
    "token_type": "Bearer",
    "expires_in": 3600,
    "context": {
      "version": 2,
      "intent": "assignment",
      "parameters": {
        "redirect_uri": "http://localhost:3000/signup"
      },
      "class": {
        "name": "Class Companion Test Course",
        "description": null,
        "state": "active",
        "picture_url": null,
        "created_date": "2024-05-09T13:49:34.125Z",
        "updated_date": "2024-05-09T13:49:34.125Z",
        "locale": null,
        "time_zone": "America/Chicago",
        "subjects": [],
        "grade_levels": [],
        "properties": {},
        "identifiers": [
          {
            "type": "canvas_id",
            "value": "338"
          }
        ],
        "product_ids": [],
        "id": "58929ea4-c30d-4989-a0aa-58550a0a4a9b",
        "school_id": "e04c615b-1ed2-4b4f-a1c4-53006cf767b3",
        "course_id": "ac34d302-0e42-4740-869b-957b425ef011",
        "subject_id": null,
        "department_id": null,
        "session_ids": [
          "90e85d1f-318d-40aa-b719-5c0d7cd33cda"
        ],
        "period_ids": [],
        "periods": [],
        "room_ids": []
      },
      "assignment": {
        "id": "95550d5a-cdca-479c-985e-d43c26935892"
      }
    },
    "type": "person"
  }
```

**Comment by jaronpate** [comment]
Hi @cpetzold! Thank you for the report.

Some of these properties do exist but are used internally. It looks like though I forgot to add the default properties like `expires_in` and `context` (from LTI launches) that the API provides. You should know however that `token_type` and `type` are different. One is from the OAuth2 spec and the other is an Edlink defined enum respectively.

I have corrected the `TokenSet` type to better represent to response you get from the grant function. You can find this fix in the `1.0.10` release.

**Comment by cpetzold** [comment]
Thanks for the quick response! I see now that I missed `type` at the bottom of the response there, I'll make sure we don't conflate the two.
